pkg/cedar/intake/translation/system_intake.go

```diff
@@ -87,23 +86,9 @@ func (si *TranslatableSystemIntake) CreateIntakeModel() (*wire.IntakeInput, erro
 		return nil, err
 	}
 
-	closedStatuses, err := models.GetStatusesByFilter(models.SystemIntakeStatusFilterCLOSED)
-	if err != nil {
-		return nil, err
-	}
-
-	status := inputStatusInitiated
-	for _, stat := range closedStatuses {
-		if si.Status == stat {
-			status = inputStatusFinal
-			break
-		}
-	}
-
 	result := wire.IntakeInput{
-		ClientID:     pStr(si.ID.String()),
-		Body:         pStr(string(blob)),
-		ClientStatus: statusStr(status),
```

Comment:
Per my previous comment -- I _think_ that removing the `ClientStatus` will break the integration.

I'm going to review the rest of the PR assuming that this integration can be addressed in some other way, but leave that to me and the CEDAR team